contracts/ClearingHouseConfig.sol

```diff
@@ -63,9 +63,6 @@ contract ClearingHouseConfig is IClearingHouseConfig, SafeOwnable, ClearingHouse
     }
 
     function setTwapInterval(uint32 twapIntervalArg) external onlyOwner {
-        // CHC_ITI: invalid twapInterval
-        require(twapIntervalArg != 0, "CHC_ITI");
```

Comment:
If we want to set _twapInterval to zero, that means the funding payment will be `(marketTwap(0)-indexTwap(0)) / duration`. Is it expected?

Reply:
As discussed, yes, it's expected. If we do not reduce the market twap interval together, it may happen that the founding rate is opposite to the expected situation. For example, the founding rate is positive, but current index price is less than market price. In this way, traders have no incentive to open long positions to push market price to be close to index price.